datasync/chaindatafetcher/chaindata_fetcher.go

```diff
@@ -233,53 +250,56 @@ func (f *ChainDataFetcher) SetComponents(components []interface{}) {
 	}
 }
 
-func (f *ChainDataFetcher) handleRequest() {
-	f.wg.Add(1)
-	defer f.wg.Done()
-	for {
-		select {
-		case <-f.stopCh:
-			logger.Info("handleRequest is stopped")
+func (f *ChainDataFetcher) handleChainEvent(reqType requestType, ev blockchain.ChainEvent) {
```

Comment:
What it actually does is `handleRequest`, not `handleChainEvent`.

Reply:
Renamed `handleChainEvent` to `handleRequestByType`. bec68e0